### PR TITLE
Use StandardType names

### PR DESCRIPTION
**Pull request by zhenxiao**
follow https://github.com/facebook/presto/pull/4325
@cberner @martint

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/type/ArrayType.java [hunk 1]
```diff
@@ -204,6 +205,6 @@ public BlockBuilder createBlockBuilder(BlockBuilderStatus blockBuilderStatus, in
     @Override
     public String getDisplayName()
     {
-        return "array<" + elementType.getDisplayName() + ">";
+        return StandardTypes.ARRAY + "<" + elementType.getDisplayName() + ">";
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/type/ArrayType.java]: static import

File: presto-main/src/main/java/com/facebook/presto/type/RowType.java [hunk 2]
```diff
@@ -47,7 +48,7 @@
     public RowType(List<Type> fieldTypes, Optional<List<String>> fieldNames)
     {
         super(new TypeSignature(
-                        "row",
+                        StandardTypes.ROW,
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/type/RowType.java]: static import

**Comment by zhenxiao** [comment]
thank you @dain get it updated

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**